utils/common.py

```diff
@@ -0,0 +1,100 @@
+import copy
+import datetime
+import functools
+import logging
+import re
+
+# import time
```

Comment:
Lets remove unwanted line if not required. Or lets keep NOTE  if required in near future.